omniauth-gplus.gemspec

```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'kramdown'
   s.add_development_dependency 'pry'
   s.add_development_dependency 'coveralls'
-  s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop', '0.24.1'
```

Comment:
@samdunne In order to pass all travis builds. The next one requires ruby's version more or equal 1.9.3, and build for 1.9.2 always failed. https://rubygems.org/gems/rubocop/versions/0.25.0